packages/hash/playwright/tests/entity-creation.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { loginUsingTempForm } from "./shared/login-using-temp-form";
+import { resetDb } from "./shared/reset-db";
+import { expect, Locator, Page, test } from "./shared/runtime";
+
+const getCellText = async (
+  canvasLocator: Locator,
+  colIndex: number,
+  rowIndex: number,
```

Comment:
There is no conventional approach to indexing rows and columns. The numbers can be zero-based, one-based or even mixed. Happy for us to use any approach, but it'd be great to add a small JSDoc as a hint.

Example (this may be incorrect):

```suggestion
  /** zero-based (column A → 0) */
  colIndex: number,
  /** zero-based (header row → 0) */
  rowIndex: number,
```

Reply:
Updated here https://github.com/hashintel/hash/pull/1790/commits/29548ceb30f92b07c05ef2c19cce6daf2ad59f94